provider/datagen/src/segmenter/transform.rs

```diff
@@ -527,12 +526,15 @@ impl SegmenterRuleProvider {
         let mut break_state_table = vec![UNKNOWN_RULE; rule_size];
 
         for rule in &segmenter.rules {
-            let break_state;
-            if let Some(state) = rule.break_state {
-                break_state = if state { BREAK_RULE } else { KEEP_RULE };
+            let break_state = if let Some(state) = rule.break_state {
+                if state {
+                    BREAK_RULE
+                } else {
+                    KEEP_RULE
+                }
             } else {
-                break_state = NOT_MATCH_RULE;
-            }
+                NOT_MATCH_RULE
+            };
```

Comment:
Nit: I think this would be clearer with a `match`. Not this PR's problem though.